src/styles/components/collapsible-error-message.less

```diff
@@ -0,0 +1,54 @@
+
+//
+// The CollapsibleErrorMessage is a component used on top
+// of forms or on other places to display a main error
+// message and a floating drop-down with the error details
+//
+.collapsible-error-message {
+
+  //
+  // Shared styling between the two items
+  //
+  .collapsible-fixed, .collapsible-floating > div {
+    background: color-lighten(@red, 90);
+    padding: (@base-spacing-unit / 2) @base-spacing-unit;
```

Comment:
In this case, this is not really used for per-screen-size spacing unit. It's more like a reference value for an small spacing around the label.

When I try the `proper` spacing implementation, it looks kind of weird on large screen sizes:
![image](https://cloud.githubusercontent.com/assets/883486/17435078/a8077af2-5b0e-11e6-8c7e-c8ad9d29205e.png)